uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader.php

```diff
@@ -188,6 +189,30 @@ private function createDefaultRelationshipTypes() {
     }
   }
 
+  /**
+   * Making All Currencies Available for new installations
+   */
+  private function makeAllCurrenciesAvailable() {
+    $dao = CRM_Core_DAO::executeQuery('SELECT * from civicrm_currency');
+    while ($dao->fetch()) {
```

Comment:
Also since the currency records might be up to 200(on my local installation), wondering if we could optimize this by using a single API call to get all enable currencies names.

```php
$result = civicrm_api3('OptionValue', 'get', array(
  'sequential' => 1,
  'return' => array("name"),
  'option_group_id' => "currencies_enabled",
));
```
Then in this loop, we can check if 
```php
if(!in_array($dao->name, $enabledCurrency){
// create the currency
}
```